s3/s3_test.go

```diff
@@ -17,7 +17,7 @@ func TestNewS3Client(t *testing.T) {
 		SecretKey: "secret",
 		Trace:     true,
 	}
-	s3If, err := NewS3Client(opts, context.Background())
+	s3If, err := NewS3Client(context.Background(), opts)
```

Comment:
this test fails, not fixing that, just lint

Reply:
Thanks